Log/logfile.log

```diff
@@ -0,0 +1,686 @@
+2015/08/16 17:50:37 INFO ######################## - Info: Start Audit.pl - ######################## 
```

Comment:
You might want to add a .gitignore in order to avoid this file in git, especially if it grows ;).